ChangeLog.md

```diff
@@ -1,10 +1,150 @@
 ##2016.08.03 version 2.0.0
+* Removal of Force parameters
```

Comment:
As first bullet: this release contains breaking changes see [the migration guide]: link to your document for details on the changes and how existing scripts are impacted.